### PR TITLE
[tw] Localize `Edge Computing` for Traditional Chinese

### LINKED ISSUE #2513: [tw] Localize `Edge Computing` for Traditional Chinese
- Localize `Edge Computing` in Traditional Chinese
- https://glossary.cncf.io/edge-computing/

### PR DESCRIPTION
**Pull request by tico88612**
### Describe your changes

Origin content from https://glossary.cncf.io/edge-computing/

### Related issue number or link (ex: `resolves #issue-number`)

resolves #2513

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/zh-tw/edge-computing.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+title: 邊緣運算
+status: Completed
+category: Technology
+---
+
+## 是什麼 {#what-it-is}
+
+邊緣運算是個[分布式系統](/zh-tw/distributed-systems/)，它將一些儲存和運算資源從主要資料中心轉移到資料來源。
```
Comment (johnlinp) [inline content/zh-tw/edge-computing.md:9]: 台灣的用法是 "分散式系統" ，而不是 "分布式系統" 。

File: content/zh-tw/edge-computing.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+邊緣計算出的結果會被送回主要資料中心做進一步處理。
+邊緣運算的例子包括手腕上的小配件或分析交通流量的電腦。
+
+## 解決的問題 {#problem-it-addresses}
+
+過去十年中，我們可以看到越來越多的邊緣裝置（例如：手機、智慧型手錶、感測器）。
+在某些情況下，即時資料處理不僅是一個不錯的選擇，而且極其重要。
+想想自動駕駛的汽車。
+現在想像一下，汽車感測器的資料必須先傳送到資料中心進行處理，然後再送回汽車，這樣汽車才能作出適當的反應。
+固有的網路延遲會是致命的。
```
Comment (johnlinp) [inline content/zh-tw/edge-computing.md:21]: 我會把這一句的 "inherent" 理解成 "由於資料傳來傳去而必然產生的" 的意思，如果直接翻成 "固有的" 好像有點不太通順。不知道如何翻成這樣子如何呢？

```
如此產生的網路延遲會是致命的。
```

原文:

> The inherent network latency could be fatal.

File: content/zh-tw/edge-computing.md (hunk 1 above)
Comment (tico88612) [reply content/zh-tw/edge-computing.md]: My fault.  
Fixed.

File: content/zh-tw/edge-computing.md (hunk 2 above)
Comment (tico88612) [reply content/zh-tw/edge-computing.md]: 我其實也覺得「固有」會有點怪。這樣翻譯可以接受。

**Review by johnlinp — APPROVED** [review]
謝謝!

**Review by hwchiu — APPROVED** [review]
感謝

**Pull request merged**